parlai/crowdsourcing/utils/analysis.py

```diff
@@ -62,6 +82,44 @@ def get_results_path_base(self) -> str:
             f'{self.__class__.__name__}__{now.strftime("%Y%m%d_%H%M%S")}',
         )
 
+    def get_worker_name(self, worker_id: str) -> str:
+        """
+        Gets the global (AWS) id of a worker from their Mephisto worker_id.
```

Comment:
Hmm, the ID won't relate to AWS here if we're not using MTurk for our Mephisto job